src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -78,6 +78,7 @@
     <ProjectReference Include="..\..\Features\ExternalAccess\AspNetCore\Microsoft.CodeAnalysis.ExternalAccess.AspNetCore.csproj" />
     <ProjectReference Include="..\ExternalAccess\VisualDiagnostics\Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.csproj" />
     <ProjectReference Include="..\..\Tools\ExternalAccess\Xaml\Microsoft.CodeAnalysis.ExternalAccess.Xaml.csproj" />
+    <ProjectReference Include="..\ExternalAccess\CompilerDeveloperSDK\Microsoft.CodeAnalysis.ExternalAccess.CompilerDeveloperSDK.csproj" />
```

Comment:
adding so @333fred doesn't have to ship this.  Matches how XAML works as well.  Only 21kb so not concerned about size.

Reply:
Do we know what will happen with the current version of my extension, where I'm including the EA dll with my extension? Will it just not be loaded?

Reply:
@333fred  Both get loaded (in different ALCs) which is not ideal.  However everything currently works, and I'm otherwise unsure how we could stage this easily.  So it seems OK for a short time to load both until you're able to update your extension.

![image](https://github.com/user-attachments/assets/b5b29ec4-29cc-42a6-9b9c-610c9fe5cf50)

Reply:
Sounds good, thanks.